### DISCUSSION TITLE
Upcoming Webinar 10/20/2021

### DISCUSSION THREAD

**Discussion by shaselton-usds** [CMSgov/price-transparency-guide#211] [General]
Please join the Centers for Medicare & Medicaid Services (CMS) on October 20th, 2021 for the first in a series of technical implementation webinars for the in-network negotiated rate and out-of-network historical billed charges machine readable file requirements in the Transparency in Coverage (TiC) Final Rules (85 FR 72158).

During the first webinar, the CMS team will cover GitHub basics and a timeline for a finalized TiC schema. CMS will also respond to appropriate questions that are posted in this announcement as well as in the Q&A chat box during the webinar.

Please register for the webinar [here](https://cms.zoomgov.com/webinar/register/WN_miwIb8_dQuueWDHuXj60gQ).

Post questions below and like/upvote questions so the team knows what’s most important to you.

**Comment by Data-san** [comment]
Thank you! Yes, this is a critical question/issue given recent updates => timeline for a finalized TiC schema?

**Comment by pjboardman** [comment]
In regards to the public discoverability, are there plans to publish the URLs to machine-readable files similar to what is done here: https://www.cms.gov/CCIIO/Resources/Data-Resources/marketplace-puf

**Reply by justplainjess1** [reply]
Second this question, please.

**Comment by mrvyhrvy** [comment]
In regards to reporting the In Network MRF when the negotiated rates are a percentage of the billed charges and we must produce the rate as a dollar amount.  There have been outstanding issues in GitHub on this specific example #129 but we are unable to identify an approved solution for this issue.  Can you please provide direction?  https://github.com/CMSgov/price-transparency-guide/issues/129

**Reply by tdfow** [reply]
also referenced in #23

**Reply by ryan-thomas-peterson** [reply]
Can we provided an 'estimated charge' in the description field to amplify the inherent assumptions we're likely forced to make in producing these files to accomodate?

**Comment by RLTx1391** [comment]
Is there any guidance on how to proceed in situations where GitHub is inconsistent with the final rule? For example, the final rule requires plain language descriptions for both the in network and allowed amount files, whereas Github guidance seems to be that it is initially going to be optional for in-network and going to be required for the allowed amount file.

**Comment by vijayanandramamoorthy** [comment]
Hi, Can we get guidance on the Service_code field for non-professional services?
As mentioned in #105, Place of service is applicable only for professional services.
There is no industry standard way of translating a type of bill into a place of service.

thank you

**Comment by ghost** [comment]
Can we also address the post from @vijayanandramamoorthy In-Network file for plans with national networks. #222

**Comment by tdfow** [comment]
Any updates on how to handle the Severity of Illness for APR-DRG #190

**Comment by tdfow** [comment]
Can CMS address the discussion on when a Provider is using SSN versus TIN, is there any masking rules that can be applied to protect the Provider SSN information?   Also, if the Provider's TIN is an SSN, then would we need to format accordingly in the MRFs?

**Reply by Ginger228** [reply]
This was also posted in the discussion #73

**Comment by jltang-kp** [comment]
@shaselton-usds Some questions from our organization:

- How will the file naming convention change after the plan names can be put up as an array. 

- Is there any field( or plan to introduce ) a field to convey payer specific information about pricing determinants for a provider in the TIP files schema.   

- Will there be a parser or interface to consume these MRFs? I cannot imagine a consumer of the file will look at it raw.

**Reply by ryan-thomas-peterson** [reply]
WRT the last bullet, even a tool to run a file through to determine the state of it's 'compliance' would be helpful.

**Comment by Devyandu** [comment]
Guidance needed on file naming convention now that a single file can have multiple plans.
If a file has 5 plans then which one of these will be in the file name? 
`<YYYY-MM-DD>_<payer or issuer name>_<plan name>_<file type name>.<file extension>"`

**Comment by tdfow** [comment]
There have been been many discussion topics open for months that are just now resulting in CMS schema updates and/or guidance.  In the past two weeks, we have seen needed schema changes, but the delay in CMS guidance has impacted schedules, designs.    What is CMS doing to limit these continued impacts?  It seems the 7/1/22 date did not consider the open discussion topics or needed guidance.  Does CMS have a timeline on when guidance will be provided for the open items?

**Comment by Data-san** [comment]
CMS - Please address tdfow's comments tomorrow. They capture well the
carryover effects with various issues, schema changes, and perhaps more
schema changes. In short, clarify the timeline to a final schema. Thank you.

**Comment by tdfow** [comment]
Did the Cost estimates published by CMS include the hosting costs for these very large files?  Are we allowed to impose any protections against entities either maliciously or not, spiking egress fees by downloading  files multiple times, etc.  
For example, can we require that the files are available upon request?

**Reply by ghost** [reply]
Great call out as that is a major concern

**Reply by tparency** [reply]
> Did the Cost estimates published by CMS include the hosting costs for these very large files? Are we allowed to impose any protections against entities either maliciously or not, spiking egress fees by downloading files multiple times, etc. For example, can we require that the files are available upon request?

on [Page 84 ](https://www.govinfo.gov/content/pkg/FR-2020-11-12/pdf/2020-24591.pdf)of the final rule:

> The Departments also do not agree that the volume of data would make the
machine-readable files too large to post publicly, regardless of whether the data is posted in two or three files. 

> The Departments’ rough estimate of file size, based, in part, upon numbers provided 
by commenters, suggests a file size of approximately 5 gigabytes.204 CMS currently makes available for download on its website some large public use file (PUF) data sets that are several gigabytes.

These assumptions by CMS are incorrect, as most plans rapidly discovered. They did not expect more than a 5gb file.

**Reply by ghost** [reply]
We are looking at potential 60TB per our MRF 1 In-Network Rates files.

**Reply by vijayanandramamoorthy** [reply]
> We are looking at potential 60TB per our MRF 1 In-Network Rates files.

Are these estimates based on the latest schema, with the service_code array and provider array updates?
thanks

**Reply by jltang-kp** [reply]
We are also attempting to generate an MRF1 and our system kicked off generating the file October 9th and is still compiling as of today. We know we are running this in a test environment so performance is throttled however we expect the MRF1 to be in the same 60tb file size range.

Honestly this does not sound feasible for a consumer to be able to download these files on a monthly basis from multiple payers and be able to fully ingest all this information.

**Reply by ghost** [reply]
Yes

**Reply by ghost** [reply]
@jltang-kp Yes that is another concern we have will be the processing time and any need to supplement our productional environment.

**Reply by Paul-Sousa** [reply]
@BobSyracuse @jltang-kp if you don't mind sharing, how many Billing Codes and TINs are you using in your estimates?  Our current estimate is so far under what you both have come up with that I'm nervous we may have missed something, or if it's just a massive difference in the sizes of our provider networks.

**Reply by shaselton-usds** [reply]
@BobSyracuse Would you mind emailing PriceTransparencyinCoverage@cms.hhs.gov about this -- the amount of data you're talking about it is extremely high. Would love to learn more about the scenario.

**Reply by ghost** [reply]
@shaselton Yes, I can....it is resulting from the ruling requiring certain associations to publish there fellow member's data along with their own. It is why it would be very helpful to have a single reference file.

**Reply by ghost** [reply]
@shaselton-usds Also, @vijayanandramamoorthy explained it very well on his post In-Network file for plans with national networks. #222 that calls out the use case that will create such a large file size.

**Reply by ghost** [reply]
Also, that 60TB was uncompressed, compressed it is estimated to be 12TB @shaselton-usds

**Comment by jonberkery-work** [comment]
Thank you for raising this concern @tdfow.  I also have these same concerns around malicious actors, the size of this data is detrimental to our network traffic.  Secondly, I am also curious has anyone estimated how often these files will be downloaded by a file consumer.  That seems like an unknown no one has been able to effectively predict as of yet.

**Comment by ericsilvertx** [comment]
I have a feeling a 1 hour meeting will not be sufficient.

**Reply by RLTx1391** [reply]
Could CMS provide a timeline of future webinars, since it was mentioned this will be a series? It'll help us with planning.

**Reply by Data-san** [reply]
My calendar only shows a 30 minute call. Sure hope that's not correct.

**Reply by tparency** [reply]
> My calendar only shows a 30 minute call. Sure hope that's not correct.

According to the invite received after registration, it's 1hr.

**Comment by ericsilvertx** [comment]
https://github.com/CMSgov/price-transparency-guide/discussions/215  

Previously posted but how do you deal with the allowed amount for the same provider if they have variation on its cost.  A provider may end up for the procedure code and location getting paid a range of value.  Do we output the average, the median or the maximum value?

**Comment by lorrainefletcher** [comment]
regarding the JSON format in GitHub for Prescription Drug File.
•	Field names in CMS 9915-F Appendix 4 do not match with GitHub Rx file schema – which fieldnames should be used when making the data available? 
o	Example :
Appendix 4:                                                “Name of Reporting Entity”
GitHub JSON schema and examples:        “reporting_entity_name”
•	The sequence of the data elements in Appendix 4 is different to GitHub. Can CMS confirm that we should use the GitHub schema sequence which flows easier in the JSON format?
o	Example:
Appendix 4 sequence                                     Plan, Provider, Drug, Pricing
GitHub JSON sequence -                                  Plan, Drug, Prices, Pharmacies
•	Historical Net Price Allocation Reporting Period definition varies between Appendix 4 and GitHub. Can CMS confirm that GitHub definition should be used and is a required field?
o	Example:
Appendix 4: “If the historical net price concessions are not known to the plan or issuer on the last publication date of the file, then the time period prior to the current reporting period used for reporting purposes must be displayed.” This implies that if the price concessions are known then this field should be blank.
GitHub: “The date in which the reporting period for the historical_net_price ended”. Required field. 
•	In GitHUB, Pharmacies array is not required however the definition states ”A list of different pharmacies objects that have specific negotiated rates for the specific NDC”  however under Pharmacies Object both the pharmacy_id_type and pharmacy_ids are required. Please confirm if Pharmacies Object are required for all drugs with negotiated rates.
•	Recent changes in GitHub including “adding the ability to include multiple plans into a single file when …” and “ updating examples with multiple plans per file” are only showing for the Allowed Amounts and In Network Files, will the same allowances be made on the Prescription Drug File?

**Comment by Ginger228** [comment]
Has CMS revisited the capitated rates.  It isn't clear how this assist a member selecting one an insurance plan.  Providers that received capitation do not send in claims for adjudication- they received monthly payments.  Capitation also not a 1:1 relationship with a procedure code.  Multiple codes could be for the PMPM. 
Example-  99075-99429, 99444,99450-99600, 99601-99602 have the same PMPM (based on member's age)
